src/current/v23.1/sql-feature-support.md

```diff
@@ -192,7 +192,7 @@ XML | ✗ | Standard | XML data can be stored as `BYTES`, but we do not offer XM
  Window functions | ✓ | Standard | [Window Functions documentation]({% link {{ page.version.version }}/window-functions.md %})
  Common table expressions | Partial | Common Extension | [Common Table Expressions documentation]({% link {{ page.version.version }}/common-table-expressions.md %})
  Stored procedures | ✗ | Common Extension | Execute a procedure explicitly. [GitHub issue tracking stored procedures support](https://github.com/cockroachdb/cockroach/issues/17511).
- Cursors | ✗ | Standard | Traverse a table's rows. To support a cursor-like use case, see the example in [Paginate Results]({% link {{ page.version.version }}/pagination.md %}).
+ Cursors | ✓ | Standard | [Cursors documentation]({% link {{ page.version.version }}/cursors.md %})
```

Comment:
Maybe cursors should be partial? Some functionality still isn't implemented (e.g. [moving backward](https://github.com/cockroachdb/cockroach/issues/77102) and [keeping open after the transaction closes](https://github.com/cockroachdb/cockroach/issues/77101)).